packages/vector_graphics/lib/src/listener.dart

```diff
@@ -6,33 +6,17 @@ import 'dart:ui' as ui;
 import 'dart:typed_data';
 
 import 'package:flutter/cupertino.dart';
-import 'package:flutter/rendering.dart';
 import 'package:vector_graphics_codec/vector_graphics_codec.dart';
 
 const VectorGraphicsCodec _codec = VectorGraphicsCodec();
 
 /// The deocded result of a vector graphics asset.
 class PictureInfo {
   /// Construct a new [PictureInfo].
-  PictureInfo._(this._handle, this.size);
+  PictureInfo._(this.picture, this.size);
 
-  /// A [LayerHandle] that contains a picture layer with the decoded
-  /// vector graphic.
-  final LayerHandle<PictureLayer> _handle;
-
-  /// Retrieve the picture layer associated with this [PictureInfo].
-  ///
-  /// Will be null if info has already been disposed.
-  PictureLayer? get layer {
-    return _handle.layer;
-  }
-
-  //// Dispose this picture info.
-  ///
-  /// It is safe to call this method multiple times.
-  void dispose() {
```

Comment:
Is it worth keeping this so we can null out the picture after?  Although picture now has a debug disposed property.  Maybe just add some asserts when we draw the picture that it's not disposed.

Reply:
Its weird having an API where you have a dispose and then have another type accessible that has the same dispose. since we null out the entire picture info I think we're fine.